components/expenses/ExpenseForm.js

```diff
@@ -572,7 +572,7 @@ const ExpenseFormBody = ({
                   whiteSpace="nowrap"
                   data-cy="expense-summary-btn"
                   buttonStyle="primary"
-                  disabled={!stepTwoCompleted || !formik.isValid}
+                  disabled={!stepTwoCompleted || !formik.isValid || !hasBaseFormFieldsCompleted}
```

Comment:
I'm a bit worried about this one. The main reason why design started to hide the rest of the form was that people were missing the title input (because it doesn't look like an input) so they ended up with this disabled submit button but had no idea what was wrong. Could we keep that enabled and rely on error reporting to prevent going next?